### PR TITLE
Configure PR rate limit

### PR DESCRIPTION
**Pull request by OussamaHaff**
- [x]  PR rate-limit
 - [x] PR Check suite configuration
 - [x] PR CI Checks configuration
 - [x] Unit tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,6 +60,7 @@
     "testEnvironment": "node",
     "transform": {
       ".(ts|tsx)": "ts-jest"
-    }
+    },
+    "verbose": true
```
Comment (viniciusml) [inline package.json:64]: Do we need to include `package-lock.json` and this file?

File: src/autoupdater.ts [hunk 2]
```diff
@@ -305,6 +429,7 @@ export class AutoUpdater {
     const readyStateFilter = this.config.pullRequestReadyState();
     if (readyStateFilter !== 'all') {
       ghCore.info('Checking PR ready state');
+      ghCore.info(`PR ${JSON.stringify(pull)}`);
```
Comment (viniciusml) [inline src/autoupdater.ts:432]: Is this for debugging?

**Review by viniciusml — APPROVED** [review]
Looks good, just left a couple of questions.

File: package.json (hunk 1 above)
Comment (leonardootto) [reply package.json:64]: Not necessarily, but i include to get more debug information.

File: src/autoupdater.ts (hunk 2 above)
Comment (leonardootto) [reply src/autoupdater.ts:432]: Yes, it appear inside the github lambda execution pane.

**Pull request merged**